docs/glossary.md

```diff
@@ -194,14 +212,21 @@ Express
     A JavaScript HTTP server with a simple API to build custom applications.
     Volto uses it as its server.
 
-Server-Side Rendering (SSR)
-    When first loading any Plone page, users will get HTML markup that closely matches the final DOM structure of the React components used to render that page.
+SSR
+server-side rendering
+    When first loading any Plone page, SSR will send HTML markup that closely matches the final DOM structure of the React components used to render that page.
+
+    More generally, when a client sends a request to an SSR page, the HTML for the page is created on the server on each request.
+    SSR enables a developer to customize a website per request and per user.
+    In addition, SSR can improve performance and search engine optimization (SEO) for a website.
 
-Single Page Application (SPA)
+SPA
+single page application
     A type of JavaScript application that aims to provide a better user experience by avoiding unnecessary reloading of the browser page, instead using AJAX to load backend information.
```

Comment:
Should we define `AJAX` too? https://en.wikipedia.org/wiki/Ajax_(programming)

Reply:
Done in [f466123](https://github.com/plone/documentation/pull/1416/commits/f4661232f14259df434967d0fc224cec98f2104a)